app/assets/stylesheets/pages.css.scss

```diff
@@ -9,6 +9,22 @@
 
 .pages-footer { color: #aaa; }
 
+// details-section
+.details-section {
+  display: inline-block;
```

Comment:
Properties should be ordered background, display, margin-bottom, padding, position, width